articles/modules/ROOT/pages/using-subqueries-to-avoid-the-eager.adoc

```diff
@@ -0,0 +1,222 @@
+= Using Subqueries to Scope and Avoid Eager Behavior
+
+:slug: using-subqueries-to-avoid-the-eager
+:author: Andrew Bowman
+:neo4j-versions: 5.x, 4.4, 4.3, 4.2, 4.1
+:tags: cypher, performance, load-csv
+:category: cypher
+
+Eager operators in a query plan can be disruptive, especially when performing writes involving large amounts of data, or batch loading.
+
+If you've used `USING PERIODIC COMMIT LOAD CSV` to import data into Neo4j, it's likely at some point that you've been bitten by the Eager:
```

Comment:
using periodic commit is no longer supported in 5.x afaik, you might want to mention that?

btw. you also get Eager with call in transactions ...